tests/unit/command/test_experiments.py

```diff
@@ -274,3 +277,128 @@ def test_experiments_remove(dvc, scm, mocker, queue, clear_all):
+                            "featurize": {"max_features": 3000, "ngrams": 1},
+                            "avg_prec": 0.5843640011189556,
+                            "roc_auc": 0.9544670443829399,
+                        }
+                    }
+                },
+            }
+        }
+    },
+    "b05eecc666734e899f79af228ff49a7ae5a18cc0": {
+        "baseline": {
+            "data": {
+                "timestamp": datetime.fromisoformat("2021-08-02T16:48:14"),
+                "params": {
+                    "params.yaml": {
+                        "data": {
+                            "featurize": {"max_features": 3000, "ngrams": 1},
+                            "train": {
+                                "seed": 20170428,
+                                "n_est": 100,
+                                "min_split": 2,
+                            },
+                        }
+                    }
+                },
+                "queued": False,
+                "running": False,
+                "executor": None,
+                "metrics": {
+                    "scores.json": {
+                        "data": {
+                            "featurize": {"max_features": 3000, "ngrams": 1},
+                            "avg_prec": 0.5325162867864254,
+                            "roc_auc": 0.9106964878520005,
+                        }
+                    }
+                },
+                "name": "master",
+            }
+        },
+        "ae99936461d6c3092934160f8beafe66a294f98d": {
+            "data": {
+                "timestamp": datetime.fromisoformat("2021-08-31T14:56:55"),
+                "params": {
+                    "params.yaml": {
+                        "data": {
+                            "featurize": {"max_features": 3000, "ngrams": 1},
+                            "train": {
+                                "seed": 20170428,
+                                "n_est": 100,
+                                "min_split": 36,
+                            },
+                        }
+                    }
+                },
+                "queued": True,
+                "running": True,
+                "executor": None,
+                "metrics": {
+                    "scores.json": {
+                        "data": {
+                            "featurize": {"max_features": 3000, "ngrams": 1},
+                            "avg_prec": 0.5843640011189556,
+                            "roc_auc": 0.9544670443829399,
+                        }
+                    }
+                },
+                "name": "exp-44136",
+            }
+        },
+    },
+}
+
+
+def test_show_experiments(capsys):
+    show_experiments(
+        all_experiments, precision=None, fill_value="", iso=True, show_csv=True
+    )
+    cap = capsys.readouterr()
+    assert (
+        "Experiment,rev,typ,Created,parent,State,scores.json:"
+        "featurize.max_features,scores.json:featurize.ngrams,"
```

Comment:
In this test, some arguments appear twice in different files, it can be used for test #5989

Reply:
Add a duplicated `parent` column in `params.yaml`